### PR TITLE
AirfRANS: gc=1.5 + lighter WD (5e-3) at 4L/256d

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis
The current best (0.007264) uses WD=1e-2 with gc=1.0. We want to test whether gc=1.5 can improve upon this when paired with a lighter weight decay (WD=5e-3). At 3L/192d, gc=1.5+WD=1e-2 showed interference — the heavy regularization may fight gradient clipping. With 4L/256d, using a lighter WD=5e-3 may allow gc=1.5 to clip effectively without over-penalizing, potentially squeezing more out of the architecture.

## Instructions
Train on AirfRANS with 4L/256d architecture, gc=1.5, WD=5e-3, T_max=5, using the following command:

```bash
cd target/icml2026 && python train.py \
  --dataset airfrans \
  --num-layers 4 \
  --hidden-dim 256 \
  --use-fourier-features \
  --no-ema \
  --optimizer adamw \
  --lr 7e-4 \
  --weight-decay 5e-3 \
  --grad-clip 1.5 \
  --scheduler cosine \
  --T-max 5 \
  --epochs 50 \
  --wandb_group airfrans-gc15-wd5e3
```

Key changes vs baseline:
- `--grad-clip 1.5` (baseline: 1.0)
- `--weight-decay 5e-3` (baseline: 1e-2)
- Everything else identical to the current best

Report `val_primary/surface_mse` at end of training.

## Baseline
Current best metrics (AirfRANS):
- `val_primary/surface_mse`: **0.007264**
- Architecture: 4L/256d + Fourier, no-EMA, AdamW, lr=7e-4, WD=1e-2, gc=1.0, T_max=5
- Baseline PR: #2727 (shoya)
- Reproduce: `cd target/icml2026 && python train.py --dataset airfrans --num-layers 4 --hidden-dim 256 --use-fourier-features --no-ema --optimizer adamw --lr 7e-4 --weight-decay 1e-2 --grad-clip 1.0 --scheduler cosine --T-max 5 --epochs 50`

### CODE REVIEW

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2784 because ADVISOR: Closing — gc=1.5 is confirmed dead at 4L/256d regardless of WD setting. 5 independent confirmations..
